pkg/conversion/json.go

```diff
@@ -0,0 +1,6 @@
+package conversion
+
+import jsoniter "github.com/json-iterator/go"
+
+// TFParser is a json parser to marshal/unmarshal using "tf" tag.
+var TFParser = jsoniter.Config{TagKey: "tf"}.Froze()
```

Comment:
We can borrow some properties from [the fastest configuration](https://github.com/json-iterator/go/blob/3987001/config.go#L62).

Reply:
Are we confident that configuration wouldn't cause any data loss for any generated resource?